src/dispatch/incident/flows.py

```diff
@@ -156,6 +156,14 @@ def incident_create_resources(*, incident: Incident, db_session=None) -> Inciden
     individual_participants, team_participants = get_incident_participants(incident, db_session)
     tactical_participant_emails = [i.email for i, _ in individual_participants]
 
+    # we add any observer added in create (like new oncall participant)
+    participant_with_observer_role = participant_service.get_by_incident_id_and_role(
```

Comment:
Is it only one or could it be more than one?

Reply:
I guess it won't be a problem because any incident participants with role `Observer` returned by `get_incident_participants(incident, db_session)` are not part of the incident yet.